<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestDataWidget</class>
 <widget class="QDockWidget" name="TestDataWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>210</width>
    <height>75</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>210</width>
    <height>75</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>210</width>
    <height>75</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Test Data</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>1</number>
    </property>
    <property name="margin">
     <number>4</number>
    </property>
    <item>
     <widget class="QToolButton" name="loadTestDataButton">
      <property name="minimumSize">
       <size>
        <width>171</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>171</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Load Test Data</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="saveTestDataButton">
      <property name="minimumSize">
       <size>
        <width>171</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>171</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Save Test Data</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>loadTestDataButton</sender>
   <signal>clicked()</signal>
   <receiver>TestDataWidget</receiver>
   <slot>loadTestData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveTestDataButton</sender>
   <signal>clicked()</signal>
   <receiver>TestDataWidget</receiver>
   <slot>saveTestData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>37</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>loadTestData()</slot>
  <slot>saveTestData()</slot>
 </slots>
</ui>
